src/test/java/com/mentoring/framework/CommonAssertions.java

```diff
@@ -0,0 +1,15 @@
+package com.mentoring.framework;
+
+import com.mentoring.pageobject.NotificationContainer;
+
+public final class CommonAssertions {
+
+    private CommonAssertions () {
+    }
+
+    public static boolean hasButtonsListErrorText(NotificationContainer notificationContainer, String errorMessage) {
```

Comment:
Let's generalize a bit, because we can: just refer to NotificationText instead of ErrorText. There is no difference between the two in our locator. We can re-use this later for other type of notification assertions.
Also I'd go further with the renaming. You're using an element's name from the container in the method that has nothing to do with it's actual function/type: it's not a button we're asserting but a notification. You don't need to refer to the data type processed (list) either, because:
1. it will cause you a headache if the data type changes during a refactor
2. at this level it doesn't matter what did we process, it's "hidden" on the page object's level.

So I'd suggest name something like: isNotificationMessagePresent or didNotificationMessageWithCorrectTextAppear